frontend/app/src/components/MainMenu/styled-components.ts

```diff
@@ -80,14 +80,6 @@ export const StyledMenuItem = styled.ul<ItemProps>(
           cursor: "not-allowed",
         }
       : {
-          "&:active": {
-            backgroundColor: theme.colors.primary,
-            color: theme.colors.white,
-            outline: "none",
-            [StyledMenuItemShortcut as any]: {
-              color: theme.colors.white,
-            },
-          },
```

Comment:
This part seems to be historical and undesired since it makes it look a bit broken.